npc/instances/EndlessTower.txt

```diff
@@ -312,6 +312,33 @@ L_Enter:
 		mes "The party leader did not generate the dungeon yet.";
 		close;
 	} else {
+		switch (getbattleflag("features/pre_re_instance_mode")) {
+		case 0:
```

Comment:
This would need some adjustments then, I'd also like to see `default:` rather than `case 0:`